contracts/hardhat.config.ts

```diff
@@ -34,7 +34,9 @@ export default {
     target: 'ethers-v5',
   },
   networks: {
-    hardhat: {},
+    hardhat: {
+      allowUnlimitedContractSize: true,
+    },
```

Comment:
can you create a ticket to remove this so we don't forget

Reply:
going to remove it in this PR once I get under size